CONTRIBUTING.md

```diff
@@ -96,3 +96,11 @@ the final release.
   pull request. This will make it easier to identify which languages the pull
   request affects, allowing us to better identify appropriate reviewer, create
   a better release note, and make it easier to identify issues in the future.
+* When merging pull requests, use the "Squash and merge" option in GitHub. Use
```

Comment:
Is there a way we can disable merge commits and multiple commits that are rebased on the github project.

Reply:
We do, it's in the "Merge Button" section in https://github.com/protocolbuffers/protobuf/settings.

But in that case if someone does want to do a merge (the two cases mentioned here), they need to enable the button and then disable it again after the merge.